BioArchLinux/ibs/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: kiri <kiri@vern.cc>
+pkgname=ibs
+_pkgname=IBS
+pkgver=1.0.3
+pkgrel=1
+pkgdesc="Illustrator for biological sequences"
+arch=('any')
+url="https://ibs.renlab.org"
+license=('custom:commercial')
+depends=('java-runtime=8' 'bash')
+source=("${_pkgname}-${pkgver}.tar.gz::http://ibs.biocuckoo.org/download/IBS_1.0.3_unix_20170614.tar.gz"
```

Comment:
the download url is static, I recommend you to replace it with 
```
"${_pkgname}-${pkgver}.tar.gz::http://ibs.biocuckoo.org/download/IBS_${pkgver}_unix_20170614.tar.gz"
```
or even with 
```
"${_pkgname}-${pkgver}.tar.gz::http://ibs.biocuckoo.org/download/IBS_${pkgver}.tar.gz"
```
(and then `pkgver` becomes `1.0.3_unix_20170614`